### PR TITLE
Modify S5644: LaYC format

### PR DESCRIPTION
**Pull request by guillaume-dequenne-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S5644/python/rule.adoc [hunk 1]
```diff
@@ -11,14 +13,19 @@ Getting, setting and deleting items using square brackets requires the accessed
...
+Performing these operations on an object that doesn't have the corresponding method will result in a `TypeError`.
 
+== How to fix it
 
-=== Noncompliant code example
+Make sure that the class for which you are trying to perform item operations implements the required methods.
+
+=== Code examples
+
+==== Noncompliant code example
```
Comment (joke1196) [inline rules/S5644/python/rule.adoc:24]: As the rule is small and the example as well this could be put in a single section, I think.

**Review by joke1196 — APPROVED** [review]
LGTM! Just added a small comment.

**Review by joke1196 — APPROVED** [review]
LGTM!

**Review by jean-jimbo-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**